NEWS.md

```diff
@@ -1,4 +1,30 @@
-## 5.7.0 - Unreleased
+## 5.8.0 - Unreleased
+
+## 5.7.0 - Released (Poppy R2 2023)
+
+The primary focus of update DTO scheme and upgraded to java 17 version 
+
+[Full Changelog](https://github.com/folio-org/mod-invoice-storage/compare/v5.6.0...v5.7.0)
+
+### Breaking changes
+* ([MODINVOICE-466](https://issues.folio.org/browse/MODINVOICE-466)) - Update DTO schema with adding fiscalYearId to PUT payload
+
+### New APIs versions
```

Comment:
As I see it, we have never used this section.

Reply:
it can be useful info, whay we need to delete it?